### PR TITLE
Added two sections on JSJabber and Ember Weekend

### PR DESCRIPTION
**Pull request by Alonski**
Added two sections on JSJabber and Ember Weekend
Also added Hello :)

### CODE REVIEW

File: source/2019-05-12-the-ember-times-issue-98.md [hunk 1]
```diff
@@ -48,21 +49,20 @@ Whether you want to come enjoy the breathtaking views, make new Ember friends, o
...
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<consider adding some bold to your paragraph>
-  
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+## [Jabber away about Javascript 📻](https://twitter.com/samselikoff/status/1128351412881063937)
+A new podcast episode of [Javascript Jabber](https://twitter.com/JSJabber) has been released in which [Sam Selikoff](https://twitter.com/samselikoff) is interviewed by [AJ ONeal](https://twitter.com/solderjs) about **Ember** 🐹. They go into detail regarding how Sam got started working with Ember and a little bit about the **history of the Ember framework**.
```
Comment (jayjayjpg) [inline source/2019-05-12-the-ember-times-issue-98.md]: Maybe we could also say

```suggestion
A new podcast episode of [Javascript Jabber](https://twitter.com/JSJabber) has been released in which [Sam Selikoff](https://twitter.com/samselikoff) is interviewed by [AJ ONeal](https://twitter.com/solderjs) about **Ember** 🐹. They go into detail regarding how Sam got started working with Ember and they also talked a little bit about the **history of the Ember framework**.
```

What do you think?

**Review by jayjayjpg — APPROVED** [review]
This reads great! I left one more suggestion (feel free to apply it, if you find it useful), but otherwise this is already good to go imho - thank you for another :sparkles: writeup!

**Comment by Alonski** [comment]
What should I do regarding this error:
``` 
65:101-65:106  warning  `hosts` may be insensitive, use `presenters`, `entertainers` instead  hostesses-hosts  retext-equality
```

**Comment by jayjayjpg** [comment]
@Alonski You should be able to resolve the alex linter issue using an exception comment above the affected line that raises the error. You can also read more about it over here: https://github.com/ember-learn/ember-blog/blob/master/source/CONTRIBUTING.md#language-checking-with-alex

```
<!--alex ignore hostesses-hosts-->
This sentence raises an error when mentioning the term host or hostess.
```

File: source/2019-05-12-the-ember-times-issue-98.md [hunk 2]
```diff
@@ -48,21 +49,20 @@ Whether you want to come enjoy the breathtaking views, make new Ember friends, o
...
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<consider adding some bold to your paragraph>
-  
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+## [Do or Donut, there is no Sizzlepie 🍩](https://twitter.com/emberweekend/status/1116022639015940096)
+The [Ember Weekend Podcast](https://twitter.com/emberweekend) released an episode last month in which Chase, Robert, and Jonathan chat about various topics such as: **an EmberConf recap** (including *secret Donut* info and *Sizzlepies*! 🥧), the **Octane preview** (including a [video](https://www.youtube.com/watch?v=BV09blWlc64) by [@gavinjoyce](https://twitter.com/gavinjoyce), [Embroider](https://github.com/embroider-build/embroider), as well as an [EmberMap video "An Intro to Broccoli.js"](https://embermap.com/topics/intro-to-broccoli)
+
+[Give it a listen here 🎧](https://emberweekend.com/episodes/do-or-donut)! You can tell us, and the hosts, what you thought on the [#media channel](https://discordapp.com/channels/480462759797063690/486955806588403773/) on [Discord](https://discordapp.com/invite/zT3asNS).
```
Comment (jayjayjpg) [inline source/2019-05-12-the-ember-times-issue-98.md:68]:
```suggestion
<!--alex ignore hostesses-hosts-->
[Give it a listen here 🎧](https://emberweekend.com/episodes/do-or-donut)! You can tell us, and the hosts, what you thought on the [#media channel](https://discordapp.com/channels/480462759797063690/486955806588403773/) on [Discord](https://discordapp.com/invite/zT3asNS).
```

for [fixing the Travis build](https://travis-ci.org/ember-learn/ember-blog/builds/533695528?utm_source=github_status&utm_medium=notification)

**Comment by Alonski** [comment]
Thanks @jessica-jordan I have been afk for the past few hours

**Pull request merged**